core/src/main/java/org/elasticsearch/index/query/support/NestedInnerQueryParseSupport.java

```diff
@@ -41,7 +40,7 @@
  */
 public class NestedInnerQueryParseSupport {
 
-    protected final QueryParseContext parseContext;
+    protected final QueryCreationContext creationContext;
```

Comment:
I am not sure whether this change should be postponed to the refactoring of nested queries, what do you think?